### PR TITLE
PWN-7225 - New Swap. Analytics #1 for main fragment 

### PR DESCRIPTION
**Pull request by gslevinkov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-7225

## Description of Work

part 1

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/swap/jupiter/statemanager/handler/SwapStateSwapLoadedHandler.kt [hunk 1]
```diff
@@ -12,7 +12,7 @@ import org.p2p.wallet.swap.model.Slippage
 
 class SwapStateSwapLoadedHandler(
     private val routesRefresher: SwapStateRoutesRefresher,
-    private val swapInteractor: JupiterSwapInteractor,
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/swap/jupiter/statemanager/handler/SwapStateSwapLoadedHandler.kt:15]: was not used

File: app/src/main/java/org/p2p/wallet/swap/ui/jupiter/main/JupiterSwapPresenter.kt [hunk 2]
```diff
@@ -196,37 +197,27 @@ class JupiterSwapPresenter(
...
             null -> null
         }
         if (allTokenAAmount != null) {
+            analytics.logTokenAAllClicked(allTokenAAmount.toString())
             cancelRateJobs()
             stateManager.onNewAction(SwapStateAction.TokenAAmountChanged(allTokenAAmount))
         }
     }
 
-    private fun getTokenAAmount(state: SwapState): BigDecimal? {
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/swap/ui/jupiter/main/JupiterSwapPresenter.kt:209]: little cleanup with extracting to empty interactor

File: app/src/main/java/org/p2p/wallet/swap/ui/jupiter/main/JupiterSwapFragment.kt [hunk 3]
```diff
@@ -124,13 +127,13 @@ class JupiterSwapFragment :
     }
 
     private fun setupWidgetsActionCallbacks() = with(binding) {
-        if (openedFrom == SwapOpenedFrom.OTHER) {
+        if (openedFrom == SwapOpenedFrom.BOTTOM_NAVIGATION) {
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/swap/ui/jupiter/main/JupiterSwapFragment.kt:130]: Show keyboard if user entered from token details, top button actions view.
if swap is opened by bottom navigation, don't show the keyboard

Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/swap/ui/jupiter/main/JupiterSwapFragment.kt:130]: roger that

**Pull request merged**